core/src/test/scala/org/apache/spark/scheduler/DAGSchedulerSuite.scala

```diff
@@ -405,6 +405,15 @@ class DAGSchedulerSuite extends SparkFunSuite with LocalSparkContext with TimeLi
     runEvent(JobCancelled(jobId, None))
   }
 
+  /** Make task set success and check result. */
+  private def checkAnswer(
+    taskSet: TaskSet,
+    results: Seq[(TaskEndReason, Any)],
```

Comment:
I think the name of `results` is not appropriate here considering its usage and also confused with `this.results` below. Maybe, `taskEndInfos`?

Reply:
Let's update `results` to `taskEndInfos` for `complete` too.